### ISSUE TITLE
Getting VisibleDeprecationWarning from numpy possibly from "get_band_peak_fg" or "plot"

### ISSUE THREAD

**Issue by finetard** [fooof-tools/fooof#202]
I was following  the example given  [here](https://fooof-tools.github.io/fooof/auto_examples/analyses/plot_mne_example.html#sphx-glr-auto-examples-analyses-plot-mne-example-py)

```
# Plot the topographies across different frequency bands
fig, axes = plt.subplots(1, 6, figsize=(15, 5))
for ind, (label, band_def) in enumerate(bands):

    # Get the power values across channels for the current band
    band_power = check_nans(get_band_peak_fg(fg, band_def)[:, 1])
    # Create a topomap for the current oscillation band
    mne.viz.plot_topomap(band_power, raw.info, cmap=cm.viridis, contours=0,
                         axes=axes[ind], show=False);

    # Set the plot title
    axes[ind].set_title(label + ' power', {'fontsize' : 20})
```
python3.8/site-packages/fooof/objs/group.py:378: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  out = np.array([np.insert(getattr(data, name), 3, index, axis=1)

python = 3.8
numpy = 1.20.1

Same error can be seen in the tutorial page itself

**Comment by TomDonoghue** [comment]
Hey @finetard - thanks for the note on this. I believe this is the same issue that was fixed in #197. The line that you mention, that raises the error (378 or around there in ` fooof/objs/group`) has now been updated, and this fix will be included in the next release. For now this is just a warning, so it doesn't actually mean anything is currently going wrong!

**Comment by finetard** [comment]
If that is the case, this issue can be closed
